installer/mac/copy_x64_binary.py

```diff
@@ -22,10 +22,19 @@ def copy_x64(x64_src_path, x64_dest_path):
     if (os.path.exists(x64_dest_path)):
         shutil.rmtree(x64_dest_path)
 
+    if brave_channel == 'release':
+        channel = ''
+    else:
+        channel = ' %s' % brave_channel.capitalize()
+
     shutil.copytree(x64_src_path, x64_dest_path, symlinks=True,
         ignore=shutil.ignore_patterns('Sparkle.framework'))
     # remove conflicting files
     os.remove(os.path.join(x64_dest_path, 'Contents', 'Info.plist'))
+    os.remove(os.path.join(x64_dest_path, 'Contents', 'Frameworks',
+                           'Brave Browser%s Framework.framework' % channel,
```

Comment:
this is also something that this script should not have to know about and makes it fragile if things change in gn. It would have been better to pass the entire path from gn

Reply:
ah we only did that in the other branches https://github.com/brave/brave-core/commit/742d7c80566ace788bde4f0e265c4677ebc6c107